lib/tools.js

```diff
@@ -131,6 +131,9 @@ function isEmptyObject(obj) {
  * iobroker does not support all characters, substitte by _
  */
 function formatGa(gaName) {
+    //filter forbidden characters
+    gaName = gaName.replace(FORBIDDEN_CHARS, '_');
+    //format gaName
     return gaName.replace(/[\.\s\/]/g, '_');
```

Comment:
can then be removed

Reply:
What exactly?
formatting? you replacing space with _ but space is not a forbidden char

Reply:
ok